insights/templates/homepage.vue.j2

```diff
@@ -125,9 +125,12 @@
 
     <!-- choropleth -->
     <chart-card color="teal">
-      <chart-card-header title="Heat map"></chart-card-header>
+      <chart-card-header title="Regional Distribution by number of Grants"></chart-card-header>
       <a name="map"></a>
-      <choropleth class="base-card__content" container="ch-1" height="500px"></choropleth>
+      <choropleth class="base-card__content" container="ch-1" height="680px"></choropleth>
+      <div class="base-card__note">
```

Comment:
Could you change line to: 

```html
<div class="base-card__note">
```
to
```html
   <div>
              <hr class="separator-light">
```

Reply:
https://github.com/ThreeSixtyGiving/insights-ng/pull/79/commits/e45d7918865e5a8327ebf6881cc90b2e443ae9f0#diff-1431a822c62efe321b5a5f6e1c83db1ea23a52ee7477eafa65028f50329cc52aL203

Reply:
Done.